### PR TITLE
[1LP][RFR] Writing fixture for collection conversion

### PR DESCRIPTION
**Pull request by anurag03**
Purpose or Intent
=================
Writing fixture for Visual object for collection conversion of Time Profile. This PR is required to get PR: https://github.com/ManageIQ/integration_tests/pull/6095 merged.

{{pytest: -v -k 'test_cloud_start_page or test_cloud_grid_page_per_item or test_cloud_tile_page_per_item or test_cloud_list_page_per_item or test_infra_grid_page_per_item or test_infra_tile_page_per_item or test_infra_list_page_per_item or test_infra_report_page_per_item or test_infra_start_page or test_timezone_setting'}}

### CODE REVIEW

**Comment by anurag03** [comment]
@RonnyPfannschmidt  please look at the changes.

File: cfme/tests/configure/test_visual_cloud.py [hunk 1]
```diff
@@ -38,31 +38,36 @@
 }
 
 
+@pytest.fixture(scope="module")
+def visual():
+    return Visual
```
Comment (RonnyPfannschmidt) [inline cfme/tests/configure/test_visual_cloud.py]: **required** this bit is missing a instantiation

Comment (anurag03) [reply cfme/tests/configure/test_visual_cloud.py]: nice catch ;)

**Comment by anurag03** [comment]
@RonnyPfannschmidt : I have implemented the changes, can you please review it.

File: cfme/configure/settings.py [hunk 2]
```diff
@@ -127,6 +127,10 @@ def step(self):
 
 
 class Visual(Updateable, Navigatable):
+
+    def __init__(self, appliance=None):
```
Comment (RonnyPfannschmidt) [inline cfme/configure/settings.py]: **REQUIRED** off hand i cant see why this `__init__` is needed - my first impression is that its only replicating the one of `Navigatable` please explain why its needed or remove it

Comment (anurag03) [reply cfme/configure/settings.py]: Ok

Comment (anurag03) [reply cfme/configure/settings.py]: @RonnyPfannschmidt : I am initialising appliance with help of that and passing it to `Navigatable.__init__`

**Comment by mshriver** [comment]
@anurag03 you need to update your email/signature to get verified commits, please. Will merge once they're verified, and will squash if you don't :)

**Comment by anurag03** [comment]
@mshriver : Done with the requested changes.

**Pull request merged**